pkg/openapiconv/convert.go

```diff
@@ -0,0 +1,270 @@
+				v3Schema.Items.Schemas = append(v3Schema.Items.Schemas, *ConvertSchema(&s))
+			}
+		}
+	}
+
+	if v2Schema.AdditionalProperties != nil {
+		v3Schema.AdditionalProperties.Schema = ConvertSchema(v2Schema.AdditionalProperties.Schema)
+	}
+	if v2Schema.AdditionalItems != nil {
+		v3Schema.AdditionalProperties.Schema = ConvertSchema(v2Schema.AdditionalProperties.Schema)
+	}
+
+	return &v3Schema
+}
+
+func ConvertSecurityScheme(v2securityScheme *spec.SecurityScheme) *spec3.SecurityScheme {
+	if v2securityScheme == nil {
+		return nil
+	}
+	securityScheme := &spec3.SecurityScheme{
+		VendorExtensible: v2securityScheme.VendorExtensible,
+		SecuritySchemeProps: spec3.SecuritySchemeProps{
+			Description: v2securityScheme.Description,
+			Type:        v2securityScheme.Type,
+			Name:        v2securityScheme.Name,
+			In:          v2securityScheme.In,
+		},
+	}
+
+	if v2securityScheme.Flow != "" {
+		securityScheme.Flows = make(map[string]*spec3.OAuthFlow)
+		securityScheme.Flows[v2securityScheme.Flow] = &spec3.OAuthFlow{
+			OAuthFlowProps: spec3.OAuthFlowProps{
+				AuthorizationUrl: v2securityScheme.AuthorizationURL,
+				TokenUrl:         v2securityScheme.TokenURL,
+				Scopes:           v2securityScheme.Scopes,
+			},
+		}
+	}
+	return securityScheme
+}
+
+func ConvertPaths(v2Paths *spec.Paths) *spec3.Paths {
+	if v2Paths == nil {
+		return nil
+	}
+	paths := &spec3.Paths{
+		VendorExtensible: v2Paths.VendorExtensible,
+		Paths:            make(map[string]*spec3.Path),
+	}
+
+	for k, v := range v2Paths.Paths {
+		paths.Paths[k] = ConvertPathItem(v)
+	}
+	return paths
+}
+
+func ConvertPathItem(v2pathItem spec.PathItem) *spec3.Path {
+	path := &spec3.Path{
+		Refable: v2pathItem.Refable,
+		PathProps: spec3.PathProps{
+			Get:        ConvertOperation(v2pathItem.Get),
+			Put:        ConvertOperation(v2pathItem.Put),
+			Post:       ConvertOperation(v2pathItem.Post),
+			Delete:     ConvertOperation(v2pathItem.Delete),
+			Options:    ConvertOperation(v2pathItem.Options),
+			Head:       ConvertOperation(v2pathItem.Head),
+			Patch:      ConvertOperation(v2pathItem.Patch),
+			Parameters: []*spec3.Parameter{},
+		},
+		VendorExtensible: v2pathItem.VendorExtensible,
+	}
+	for _, param := range v2pathItem.Parameters {
+		path.Parameters = append(path.Parameters, ConvertParameter(param))
+	}
+	return path
+}
+
+func ConvertOperation(v2Operation *spec.Operation) *spec3.Operation {
+	if v2Operation == nil {
+		return nil
+	}
+	operation := &spec3.Operation{
+		VendorExtensible: v2Operation.VendorExtensible,
+		OperationProps: spec3.OperationProps{
+			Description:  v2Operation.Description,
+			ExternalDocs: ConvertExternalDocumentation(v2Operation.OperationProps.ExternalDocs),
+			Tags:         v2Operation.Tags,
+			Summary:      v2Operation.Summary,
+			Deprecated:   v2Operation.Deprecated,
+			OperationId:  v2Operation.ID,
+			Parameters:   []*spec3.Parameter{},
+		},
+	}
+
+	for _, param := range v2Operation.Parameters {
```

Comment:
`Parameter` is `Refable` but I do not see this case being handled.

Reply:
Fixed.